client/src/App.test.tsx

```diff
@@ -1,15 +1,177 @@
+        const { container } = renderView(
+          '/election/1/audit-board/audit-board-1'
+        )
+        await screen.findByAltText('Arlo, by VotingWorks')
+        expect(container).toMatchSnapshot()
+      })
+    })
+
+    it('renders ja logged in properly', async () => {
+      const expectedCalls = [apiMocks.successAuth(userMocks.ja)]
+      await withMockFetch(expectedCalls, async () => {
+        const { container } = renderView(
+          '/election/1/audit-board/audit-board-1'
+        )
+        await screen.findByAltText('Arlo, by VotingWorks')
+        expect(container).toMatchSnapshot()
+      })
+    })
+
+    it('renders aa logged in properly', async () => {
+      const expectedCalls = [apiMocks.successAuth(userMocks.aa)]
+      await withMockFetch(expectedCalls, async () => {
+        const { container } = renderView(
+          '/election/1/audit-board/audit-board-1'
+        )
+        await screen.findByAltText('Arlo, by VotingWorks')
+        expect(container).toMatchSnapshot()
+      })
+    })
+
+    it.skip('renders ab logged in properly', async () => {
+      const expectedCalls = [apiMocks.abAuth]
+      await withMockFetch(expectedCalls, async () => {
+        const { container } = render(
+          <MemoryRouter
+            initialEntries={['/election/1/audit-board/audit-board-1']}
+            initialIndex={0}
+          >
+            <App />
+          </MemoryRouter>
+        )
+        await screen.findByText('Audit Board #1: Member Sign-in')
+        expect(container).toMatchSnapshot()
+      })
+    })
+  })
+
+  describe('/election/:electionId/jurisdiction/:jurisdictionId', () => {
+    it('renders unauthenticated properly', async () => {
+      const expectedCalls = [apiMocks.failedAuth]
+      await withMockFetch(expectedCalls, async () => {
+        const { container } = renderView('/election/1/jurisdiction-id-1/1')
+        await screen.findByAltText('Arlo, by VotingWorks')
+        expect(container).toMatchSnapshot()
+      })
+    })
+
+    it('renders ja logged in properly', async () => {
+      const expectedCalls = [apiMocks.successAuth(userMocks.ja)]
+      await withMockFetch(expectedCalls, async () => {
+        const { container } = renderView('/election/1/jurisdiction-id-1/1')
+        await screen.findByAltText('Arlo, by VotingWorks')
+        expect(container).toMatchSnapshot()
+      })
+    })
+
+    it('renders aa logged in properly', async () => {
+      const expectedCalls = [
+        apiMocks.successAuth(userMocks.aa),
+        aaApiCalls.getRounds,
+        aaApiCalls.getJurisdictions,
+        aaApiCalls.getContests,
+        aaApiCalls.getSettings,
+        aaApiCalls.getJurisdictionFile,
+        aaApiCalls.getRounds,
+        aaApiCalls.getRounds,
+        aaApiCalls.getJurisdictions,
+        aaApiCalls.getContests,
+        aaApiCalls.getSettings,
+        aaApiCalls.getSettings,
+        aaApiCalls.getJurisdictionFile,
+      ]
```

Comment:
this seems like so many API calls just to render one screen! do you know what's going on?

Reply:
@jonahkagan This is the status box, and it has a bunch of api calls, yeah. :stuck_out_tongue_closed_eyes:

Reply:
I'm actually kind of confused by the tests in this `describe` block. I'm not sure what the URL `/election/1/jurisdiction-id-1/1` corresponds to. It looks like the pattern is `/election/:electionId/:jurisdictionId/:anotherParam`, but I can't find that in the code.

I think this is an example where the snapshot tests make it hard for me to figure out what's supposed to be going on.

Reply:
@jonahkagan Actually, that's a bug, haha. That's not what the url is supposed to be. :stuck_out_tongue_closed_eyes: It's supposed to be `/election/1/jurisdiction/jurisdiction-id-1` but my copy and paste got screwed up at some point. Good catch on that.

Ironically, when checking on this and trying to figure out if I got the mocked data right again just now, I used the snapshots heavily to know what was being rendered given the various configurations.